src/server/pause.js

```diff
@@ -0,0 +1,131 @@
+const test = window.d2lTest = {};
+test.pause = new Promise(r => test.start = r);
+
+const controls = `
+	<style>
+		#d2l-test-controls [hidden] {
+			display: none !important;
+		}
+
+		#d2l-test-controls #start, #run {
+			display: flex;
+			gap: 1em;
+			font-size: 18px;
+			font-family: 'Lato', sans-serif;
+			padding: 1em;
+			background: #fff;
+			color: #222;
+			margin: calc(-30px + -8px) calc(-30px + -8px) calc(30px + 8px);
+			flex-wrap: wrap;
+			align-items: center;
+			box-shadow: 0 0 5px rgba(0,0,0,.5);
+		}
+
+		#d2l-test-controls #start {
+			font-size: 24px;
+			height: 600px;
+			flex-direction: column;
+			justify-content: center;
+			margin-bottom: 0;
+			box-shadow: none;
+		}
+
+		#d2l-test-controls #test-name {
+			flex: 1;
+		}
+
+		#d2l-test-controls button {
+			font-family: 'Lato', sans-serif;
+			background-color: #e3e9f1;
+			color: #202122;
+			border-radius: .3em;
+			border-style: none;
+			font-weight: 700;
+			font-size: .9em;
+			margin: 0;
+			min-height: calc(2em + 2px);
+			outline: none;
+			padding: .55em 1.5em;
+			text-align: center;
+			line-height: 1em;
+			cursor: pointer;
+		}
+
+		#d2l-test-controls button:focus {
+			background-color: #cdd5dc;
+		}
+
+		#d2l-test-controls button:focus {
+			background-color: #cdd5dc;
+		}
+
+		#d2l-test-controls button:focus-visible {
+			box-shadow: 0 0 0 2px #fff, 0 0 0 4px #006fbf;
+		}
+
+		#d2l-test-controls button.primary:focus {
+			background-color: #004489;
```

Comment:
Was this intentional?
<img width="101" alt="Screenshot 2023-07-24 at 2 44 46 PM" src="https://github.com/BrightspaceUI/testing/assets/5491151/3fbf0910-3c0a-416a-827e-f42da6f073c6">

Suggest either always having the background color set on "primary" buttons and also setting the foreground to white, or just not having the concept of primary.

Reply:
Ah I see this was resolved in a follow-up PR.

Reply:
Yeah things are slightly messy trying to piece it all back together from the original big PR.